Firebase/Auth/FirebaseAuth.podspec

```diff
@@ -20,8 +20,16 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '7.0'
+  s.osx.deployment_target = '10.10'
 
   s.source_files = '**/*.[mh]'
+  s.osx.exclude_files =
```

Comment:
And FIRAuthAppCredentialManager here, too, for it is only used by Phone Number Auth.